frontend/poems/display-poems.js

```diff
@@ -14,18 +14,32 @@ function getPoems () {
   const req = { method: 'GET' }
   fetch('poems.json', req)
     .then(res => res.json())
-    .then(data => appendData(data))
+    .then(data => displayData(data))
     .catch(err => console.log('error: ' + err))
 }
 
-function appendData (data) {
-  const mainCont = document.querySelector('#poem')
-
-  for (let i = 0; i < data.length; i++) {
-    const div = document.createElement('div')
-    div.innerHTML = `${data[i].text} ...signed ${data[i].author} --
-                        ${data[i].location} -- ${data[i].datetime}
-                        -- ROUTE ${data[i].route}`
-    mainCont.appendChild(div)
+function displayData (data) {
+  
+  
+  var i = 0
+  function myLoop(){
+    let rand = Math.floor(Math.random() * data.length)
+    let text = `<b>${data[rand].poem}</b> 
+      <br><br> ...signed <b>${data[rand].author}</b> 
+      <br><br> ✩ (${data[rand].lat}, ${data[rand].long})
```

Comment:
**suggestion** 
instead of displaying the GPS coordinates, what if we create a [link to Google maps](https://stackoverflow.com/questions/1801732/how-do-i-link-to-google-maps-with-a-particular-longitude-and-latitude) at that location? something like:

```js
<a href="https://www.google.com/maps/?q=${data[rand].lat},${data[rand].long}" target="_blank">location</a>
```